library/core/src/ptr/mod.rs

```diff
@@ -15,7 +15,7 @@
 //! The precise rules for validity are not determined yet. The guarantees that are
 //! provided at this point are very minimal:
 //!
-//! * For operations of [size zero][zst], *every* pointer is valid, including the [null] pointer.
+//! * For operations of size zero, *every* pointer is valid, including the [null] pointer.
```

Comment:
This leaves a dangling `[zst]` markdown link further down this file, please also remove that.